scripts/conf.py

```diff
@@ -103,7 +103,7 @@ def conflines(cloud_parameters, lkp: util.Lookup) -> str:
         "SuspendTimeout": cloud_parameters.get("suspend_timeout", 300),
         "TreeWidth": "65533" if any_dynamic else None,
         "JobSubmitPlugins": "lua" if any_tpu else None,
-        "TopologyPlugin": "topology/tree",
+        "TopologyPlugin": cloud_parameters.get("topology_plugin", "topology/none"),
```

Comment:
IMHO,  we should default to `"topology/tree"` to preserver the behavior. Is there a good reason to do otherwise?